FiftyOne.DeviceDetection/DeviceDetectionOnPremisePipelineBuilder.cs

```diff
@@ -51,6 +52,11 @@ public class DeviceDetectionOnPremisePipelineBuilder :
 
         private bool? _autoUpdateEnabled = null;
         private bool? _dataUpdateOnStartUpEnabled = null;
+        private string _dataUpdateUrlString = null;
+        private Uri _dataUpdateUrlUri = null;
```

Comment:
wouldn't it be better to just keep the Uri and convert the string representation into Uri?